Felienne/hedy
# Disallow double submission of a correct quiz answer

**Disallow double submission of a correct quiz answer**

**Description**

_Prevent incorrect score of quiz due to re-submission of correct quiz answers_

**Fixes #2666**

**How to test**

- Run Hedy locally and go to any quiz. Submit the correct answer for question 1, so that you see the feedback page. Instead of clicking the button "Go to question 2", click the back button of your browser and submit once again the correct answer to question 1. Repeat this step with as many questions as you want and as many times as you want. When you reach the end of the quiz, check that the score does not exceed 100 and that the number of attempts for each question reflects the number of times you made a submission.
- Rerun the previous scenario for the same level.

**Checklist**
